bin/process_xenium.py

```diff
@@ -61,39 +61,44 @@ def xenium_to_anndata(
 
     if load_clusters:
         for cluster in [
-            d for d in (p / "analysis" / "clustering").iterdir() if d.is_dir()
+            d for d in (path / "analysis" / "clustering").iterdir() if d.is_dir()
         ]:
             cluster_name = cluster.name.replace("gene_expression_", "")
             cluster_df = pd.read_csv(
                 cluster / "clusters.csv",
                 index_col="Barcode",
             )
 
-            clusters = cluster_df.reindex(adata.obs.index.astype(int))
-            adata.obs[cluster_name] = pd.Categorical(clusters["Cluster"])
+            clusters = cluster_df.reindex(adata.obs.index, fill_value="Undefined")
+            adata.obs[cluster_name] = pd.Categorical(clusters["Cluster"].astype(str))
 
     if load_embeddings:
         embeddings = [
             ("umap", "2_components"),
-            ("tsne", "2_components"),
```

Comment:
Is this no longer an expected output or is it just not that useful?

Reply:
Both:
a) tSNE is not shown on our local Xenium outpus or the Example brain data ( both being xenium v1.3 )
b) tSNE and PCA aren't really useful in these cases as most people will only expect UMAP